tests/helpers.py

```diff
@@ -175,11 +181,86 @@ def run_cdo_table_cli(model_output_dir, cdo_table_file, perturbed_model_output_d
     run_cli(cdo_table, args)
 
 
-def run_cli(command, args):
+def run_select_members_cli(
+    stats_file_name,
+    selected_members_file_name,
+    tolerance_file_name,
+    test_tolerance=False,
+    max_member_num=15,
+    iterations=50,
+    max_factor=50.0,
+    log=None,
+):
+    args = [
+        "--stats-file-name",
+        stats_file_name,
+        "--selected-members-file-name",
+        selected_members_file_name,
+        "--tolerance-file-name",
+        tolerance_file_name,
+        "--max-member-num",
+        str(max_member_num),
+        "--iterations",
+        str(iterations),
+        "--max-factor",
+        str(max_factor),
+    ]
+    if test_tolerance:
+        args.append("--test-tolerance")
+    return run_cli(select_members, args, log)
+
+
+def run_cli(command, args, log=None):
+    if log:
+        log.set_level(logging.INFO)
+
     runner = CliRunner()
     result = runner.invoke(command, args)
+    if not log:
+        catch_error(result)
+    else:
+        return log.text
+
+
+def catch_error(result):
     if result.exit_code != 0:
         error_message = "Error executing command:\n" + result.output
         if result.exception:
             error_message += "\nException: " + str(result.exception)
         raise Exception(error_message)
+
+
+def create_artificial_stats_file(filename, configurations, seed, perturbation):
```

Comment:
Just a matter of taste, I would prefer
```suggestion1
def create_random_stats_file(filename, configurations, seed, perturbation):
```
or 
```suggestion2
def create_stats_file_with_random_values(filename, configurations, seed, perturbation):
```
But open for other suggestion or convincing ;)

Reply:
done